### DISCUSSION TITLE
No publishable resources for tag [webhook-client-migrations]

### DISCUSSION THREAD

**Discussion by Kamrul-Haque** [spatie/laravel-stripe-webhooks#155] [Q&A]
while running  `php artisan vendor:publish --provider="Spatie\WebhookClient\WebhookClientServiceProvider" --tag="webhook-client-migrations`, it says `No publishable resources for tag [webhook-client-migrations]` and no migration file is published

**Comment by freekmurze** [comment]
Seems like it's working here, so it must be something at your end.

Should you find that the package is in error, feel free to submit a PR to fix it.

**Reply by Kamrul-Haque** [reply]
I am using v2.6 of the package with Laravel v8 & PHP v7.4. Might be a version issue?

**Reply by kiddhex** [reply]
Try changing the tag to "migrations".
